src/components/TimeStamp.js

```diff
@@ -1,11 +1,20 @@
+import React from 'react';
 import { DateTime } from 'luxon';
+import PropTypes from 'prop-types';
 
-const TimeStamp = (props) => {
+const timeStamp = (props) => {
```

Comment:
No need to have modified this component. Also, React components by convention use capital letters as their names, so `Timestamp` was correct.